src/renderer/components/menu/menu.tsx

```diff
@@ -232,6 +233,12 @@ export class Menu extends React.Component<MenuProps, State> {
     }
   }
 
+  onBlur() {
+    if (document.activeElement?.tagName == "IFRAME") {
```

Comment:
Is this guaranteed to be upper case. Would checking the lower case be safer?

Reply:
First iteration was with `toLowerCase` checking. But it's safe while we're using `html` not `xhtml` for our markup.